core/templates/dev/head/components/VersionDiffVisualizationDirective.js

```diff
@@ -253,11 +250,11 @@ oppia.directive('versionDiffVisualization', [
             controller: [
               '$scope', '$http', '$modalInstance', '$timeout', 'newStateName',
               'oldStateName', 'newState', 'oldState', 'headers',
-              function(
-                  $scope, $http, $modalInstance, $timeout, newStateName,
-                  oldStateName, newState, oldState, headers) {
+              'explorationData', function($scope, $http, $modalInstance,
```

Comment:
Move `function` to new line. Make sure line breaks in function() correspond to line breaks in the stringified dependencies.